docs/basics/tables.md

```diff
@@ -65,7 +65,35 @@ class MyDb(override val connector: CassandraConnection) extends Database[MyDb](c
 }
 ```
 
+### The name of the table can be controlled using `NamingStrategy`.
 
+A simple import can help you control how table names are generated. Phantom offers three variants,
```

Comment:
An import controls how table names are generated. Phantom offers three variants via `com.outworkers.phantom.NamingStrategy`. These control only the `tableName`.